### PR TITLE
tests: Ability to delete workspace from the system console.

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Ref PR: https://github.com/mattermost/mattermost-webapp/pull/12077

Ref: Story: https://mattermost.atlassian.net/browse/MM-49442

demo Video: https://drive.google.com/file/d/1n6VLGclx_nb6Hr8K-JNLqcg5wOAKt6JL/view

### CODE REVIEW

**Review by ConorMacpherson — APPROVED** [review]
A few small things that might need to be changed. Overall, looks good to me!

File: data/test-cases/cloud/delete-workspace/test-case-1.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Ref PR: https://github.com/mattermost/mattermost-webapp/pull/12077
+
+Ref: Story: https://mattermost.atlassian.net/browse/MM-49442
+
+demo Video: https://drive.google.com/file/d/1n6VLGclx_nb6Hr8K-JNLqcg5wOAKt6JL/view
+
+**Prerequisites:**
+
+1. Make sure its an cloud instance and logged in with System Admin user.
```
Comment (lindalumitchell) [inline data/test-cases/cloud/delete-workspace/test-case-1.md:45]:
```suggestion
1. Make sure it is a cloud instance and logged in with System Admin user.
```

File: data/test-cases/cloud/delete-workspace/test-case-1.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+2. Navigate to System Console > Billing & Account > Subscriptions.
+3. Verify that at the bottom of the screen we can see '<Workspace name >' and **Delete Workspace** button.
+   **Note:** Only an instance with monthly subscription plan will have delete button option. i.e Cloud Professional
+4. Verify if the instance is on Yearly subscription plan, we can see **Contact Us** button.
+
+**Scenario 1:**
+Instance should be on a Free plan
+
+1. On System Console > Billing & Account > Subscriptions, Click on **Delete Workspace** button.
+2. Should chose 1 option among
```
Comment (lindalumitchell) [inline data/test-cases/cloud/delete-workspace/test-case-1.md:55]:
```suggestion
2. Should choose 1 option among
```

File: data/test-cases/cloud/delete-workspace/test-case-1.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+3. Once the workspace is deleted, you will see a success screen stating that the workspace has been deleted.
+4. Click the Go to Mattermost.com button.
+
+**Expected**
+
+**Scenario 1:**
+
+1. Should see a small survey modal is diaplayed.
+   - Title: **Please share your reason for deleting**.
+   - should have radio button to select option.
```
Comment (lindalumitchell) [inline data/test-cases/cloud/delete-workspace/test-case-1.md:72]:
```suggestion
   - Should have radio button to select option.
```

File: data/test-cases/cloud/delete-workspace/test-case-2.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Ref PR: https://github.com/mattermost/mattermost-webapp/pull/12077
+
+Ref: Story: https://mattermost.atlassian.net/browse/MM-49442
+
+demo Video: https://drive.google.com/file/d/1n6VLGclx_nb6Hr8K-JNLqcg5wOAKt6JL/view
+
+**Prerequisites:**
+
+1. Make sure its an cloud instance and logged in with System Admin user.
```
Comment (lindalumitchell) [inline data/test-cases/cloud/delete-workspace/test-case-2.md:45]:
```suggestion
1. Make sure it is a cloud instance and logged in with System Admin user.
```

**Review by lindalumitchell — APPROVED** [review]
Just commented on some minor text suggestions. Otherwise LGTM!

File: data/test-cases/cloud/delete-workspace/test-case-1.md (hunk 1 above)
Comment (yasserfaraazkhan) [reply data/test-cases/cloud/delete-workspace/test-case-1.md]: done

File: data/test-cases/cloud/delete-workspace/test-case-1.md (hunk 2 above)
Comment (yasserfaraazkhan) [reply data/test-cases/cloud/delete-workspace/test-case-1.md]: done

File: data/test-cases/cloud/delete-workspace/test-case-2.md (hunk 4 above)
Comment (yasserfaraazkhan) [reply data/test-cases/cloud/delete-workspace/test-case-2.md]: done

File: data/test-cases/cloud/delete-workspace/test-case-1.md (hunk 3 above)
Comment (yasserfaraazkhan) [reply data/test-cases/cloud/delete-workspace/test-case-1.md]: v

**Pull request merged**